extensions/wikia/Scribe/ScribeEventProducer.class.php

```diff
@@ -425,4 +437,18 @@ public function setMediaLinks( $oPage ) {
 
 		return $links;
 	}
+
+	/*
+	 * Helper methods
+	 */
+	/** @return \Wikia\Rabbit\ConnectionBase */
+	protected static function getRabbit() {
+		global $wgEditEventsRabbitConfig;
+
+		if ( !isset( self::$rabbit ) ) {
+			self::$rabbit = new ConnectionBase( $wgEditEventsRabbitConfig );
```

Comment:
Where is `wgEditEventsRabbitConfig` defined?

Reply:
Just created PR for config part: https://github.com/Wikia/config/pull/2343